chroma-manager/storage_server.repo

```diff
@@ -37,6 +37,17 @@ gpgkey=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-
 repo_gpgcheck=0
 enabled=1
 
+[wcjohnso-manager-for-lustre]
+name=Copr repo for manager-for-lustre owned by wcjohnso
+baseurl=https://copr-be.cloud.fedoraproject.org/results/wcjohnso/manager-for-lustre/epel-7-$basearch/
+type=rpm-md
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/wcjohnso/manager-for-lustre/pubkey.gpg
+repo_gpgcheck=0
+enabled=1
+enabled_metadata=1
+
```

Comment:
Per my [previous comment](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/384#issuecomment-344921040), you need to also add the above to the /usr/share/chroma-manager/storage_server.repo that is written to at [this line](https://github.com/intel-hpdd/intel-manager-for-lustre/blob/master/chroma-manager/tests/framework/integration/installation_and_upgrade/run_tests#L271) with something like:

```
cat <<EOF >> /usr/share/chroma-manager/storage_server.repo
[wcjohnso-manager-for-lustre]
name=Copr repo for manager-for-lustre owned by wcjohnso
baseurl=https://copr-be.cloud.fedoraproject.org/results/wcjohnso/manager-for-lustre/epel-7-$basearch/
type=rpm-md
skip_if_unavailable=True
gpgcheck=1
gpgkey=https://copr-be.cloud.fedoraproject.org/results/wcjohnso/manager-for-lustre/pubkey.gpg
repo_gpgcheck=0
enabled=1
enabled_metadata=1
￼ 
EOF
```